frontend/src/pages/transactions/[hash].tsx

```diff
@@ -143,13 +158,10 @@ export const getServerSideProps: GetServerSideProps<
       transactionBaseInfo.hash,
       transactionBaseInfo.signerId,
     ]);
-    const actions = transactionInfo.transaction.actions.map((action) => {
-      const kind = Object.keys(action)[0] as keyof RpcAction;
-      return {
-        kind,
-        args: action[kind],
-      };
-    });
+    console.log("transaction", transactionInfo.transaction.actions);
```

Comment:
Remove this please

Reply:
Got it
We need a linter with `no-console` rule :)